src/openrct2/world/Footpath.cpp

```diff
@@ -184,15 +182,15 @@ money32 footpath_provisional_set(int32_t type, int32_t x, int32_t y, int32_t z,
         }
         else if (
             gFootpathConstructSlope == TILE_ELEMENT_SLOPE_FLAT
-            || gFootpathProvisionalPosition.z * 8 < gFootpathConstructFromPosition.z)
+            || gFootpathProvisionalPosition.z < gFootpathConstructFromPosition.z)
         {
             // Going either straight on, or down.
-            virtual_floor_set_height(gFootpathProvisionalPosition.z * 8);
+            virtual_floor_set_height(gFootpathProvisionalPosition.z);
         }
         else
         {
             // Going up in the world!
-            virtual_floor_set_height((gFootpathProvisionalPosition.z + 2) * 8);
+            virtual_floor_set_height(gFootpathProvisionalPosition.z + 2 * 8);
```

Comment:
Please revert. This makes the order ambiguous. In standard math rules, this would be equivalent to `gFootpathProvisionalPosition.z + 16`.

Reply:
you mean `gFootpathProvisionalPosition.z + 16` i assume. It used to be a small z now its a big z so only the 2 needs the * 8.

Reply:
Ok, but then I do prefer if you put brackets around the `2 * 8` bit (for legibility and disambiguation).